cypress/e2e/tests/migration/reports/filter.test.ts

```diff
@@ -112,8 +112,13 @@ describe(["@tier2"], "Reports filter validations", () => {
         clickByText(button, clearAllFilters);
 
         //close Row Details of Identified risks
-        click(closeRowIdentifiedRisk);
-
+        cy.contains("h3", "Identified risks")
```

Comment:
Please check if there is an existing method to close the row in reports.ts or utils.

If not you can create one and just call .
report.closeRow()

Reply:
Done, created closeArticle method